internal/lattice/lattice.go

```diff
@@ -107,7 +107,7 @@ func (la *Lattice) addNode(pos, id, start int, class NodeClass, surface string)
 }
 
 // Build builds a lattice from the inputs.
-func (la *Lattice) Build(inp string) {
+func (la *Lattice) Build(inp string) error {
```

Comment:
__[gometalinter]__
* [gocyclo] cyclomatic complexity 17 of function (*Lattice).Build() is high (> 15)

<!-- Comment by [SideCI](https://sideci.com) -->